components/_global-navigation.scss

```diff
@@ -11,71 +11,81 @@ $notification-counter-size: 18px;
 	border-width: $border-top-width 0 0;
 	box-shadow: 0 1px 0 0 $color-faint-gray;
 
-	&__content-bar {
-		height: $nav-height;
-		margin: 0 auto;
-		position: relative;
-		width: 1024px - $scrollbar-width;
-		z-index: 1;
+	@media only screen and (max-width: 1023px) {
```

Comment:
1. Can we define these breakpoints in one place?
2. Did we decide to have such breakpoint?